README.md

```diff
@@ -22,11 +22,9 @@ This library has been tested on the following platforms:
 
 ## Hardware setup
 
-**Important Note:** The SPS30 requires 5V input voltage +/-0.5V in order to
-provide correct output values.
+**Important Notes:** 
 
-* When using a 3.3V based Arduino, make sure to use the appropriate voltage
-  regulators and level shifters for I2C!
+* The SPS30 requires 5V supply voltage +/-0.5V in order to provide correct output values; the SPS30 should be compatible with 3.3V I2C levels, so as long as the supply voltage is correct the SPS30 should work fine with 3.3V setups
```

Comment:
Could you wrap at col80?

Reply:
done in the second commit, will squash in merge